lib/ash_json_api/serializer.ex

```diff
@@ -206,73 +220,164 @@ defmodule AshJsonApi.Serializer do
   end
 
   defp put_count_param(query, %{count: count}) when is_integer(count) do
-    Map.update(query, "page", %{count: count}, &Map.put(&1, :count, count))
+    Map.update(query, "page", %{count: count}, &Map.put(&1, :count, true))
```

Comment:
Should this be `true`? Or should this be the `count`?

Reply:
I set it to true because when you perform a query like say:
`/posts?sort=-inserted_at&page[size]=#{page_size}&page[count]=true`

When its set to count, the links generated look like:
`prev: /posts?sort=-inserted_at&page[size]=10&page[count]=90`

which doesnt match the original query made, and also not sure what use having the count in the query params does as this should be in the meta field instead I think

Reply:
🤔 I see what you mean.We should change the first one to `%{count: true}` also then? Actually I'm not sure we need this at all.

Reply:
I think it might still be good to keep, if you make a query with count=true you probably would want subsequent page links to have that as well

Reply:
That makes sense, but the `count` needs to be the same in either case, i.e

```elixir
Map.update(query, "page", %{count: true}, &Map.put(&1, :count, true))
```

Reply:
Once this is addressed I think its good to merge!

Reply:
Oh got it, updated!